src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,74 @@
+package Duke;
+
+import java.io.*;
+import java.util.Scanner;
+
+
+public class Storage {
+    private String path;
+
+    public Storage(String path){
+        this.path = path;
+    }
+
+    public void save() {
+        try {
+            BufferedWriter writer = new BufferedWriter(new FileWriter(path));
+            String tasks = "";
+            for (Task task : TaskList.taskList) {
+                tasks += task.storeToString() + "\n";
+            }
+            writer.write(tasks);
+            writer.close();
+        } catch(IOException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public void load() throws IOException{
+
+        File file = new File(this.path);
+        file.getParentFile().mkdirs();
+        if (!file.exists()) {
+            file.createNewFile();
+        }
+        Scanner sc = new Scanner(file);
+
+
+
+        while (sc.hasNext()) {
+            String event = sc.nextLine();
+            String type = event.substring(0,1);
+            String status = event.substring(2,3);
+            Task cur = null;
+
+            switch (type) {
+                case "T":
+                    cur = new ToDo(event.substring(4));
+                    break;
+                case "D":
+                    int indexofdeadline = event.lastIndexOf("|") + 1;
```

Comment:
Should we use camel case for variable names?
```suggestion
                    int indexOfDeadline = event.lastIndexOf("|") + 1;
```